cns/ipampool/monitor.go

```diff
@@ -105,7 +104,10 @@ func (pm *Monitor) Start(ctx context.Context) error {
 			pm.metastate.batch = scaler.BatchSize
 			pm.metastate.max = scaler.MaxIPCount
 			pm.metastate.minFreeCount, pm.metastate.maxFreeCount = CalculateMinFreeIPs(scaler), CalculateMaxFreeIPs(scaler)
-			pm.once.Do(func() { close(pm.started) }) // close the init channel the first time we receive a NodeNetworkConfig.
+			pm.once.Do(func() {
+				pm.spec = nnc.Spec // set the spec from the NNC initially (afterwards we write the Spec so we know target state).
```

Comment:
Can you add a log line here

Reply:
Also add a log line in func (pm *Monitor) Update(nnc *v1alpha.NodeNetworkConfig) error { function.